basiclibrary/src/main/java/Library.java

```diff
@@ -0,0 +1,50 @@
+import java.util.Random;
+
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+public class Library {
+
+    public static int[] roll(int rolls) {
+        int[] values = new int[rolls];
+        Random rnd = new Random();
+        for (int i = 0; i < values.length; i++) {
+            int roll = rnd.nextInt(6) + 1;
+            values[i] = roll;
+        }
+        return values;
+    }
+
+    public static boolean containsDuplicates(int[] arr) {
+        boolean hasDupe = false;
+        outer: for (int i = 0; i < arr.length -1; i++) {
```

Comment:
Why are you adding "outer" here? Did you see this in documentation? Is this a comment? If a comment, then where are your //?

Reply:
This was something mentioned in one of our readings about loops. There is the option to name for loops so that you can specify which loop a break statement breaks out of. I utilized this to ensure my break statement completely ended both my loops, which I used instead of a return since I generally write my functions to only include one return statement.